lib/builder-webpack4/src/index.ts

```diff
@@ -39,6 +39,13 @@ export const executor = {
   get: webpack,
 };
 
+export const makeStatsFromError: (err: string) => Stats = (err) =>
+  ({
+    hasErrors: () => true,
+    hasWarnings: () => false,
+    toJson: () => ({ warnings: [] as any[], errors: [err] }),
+  } as any);
```

Comment:
Is this something we want to export?

In the future we'll want to map the builders internal stats object to something common for all builders.
Right now we're leaking webpack details from our 2 builders. This is a known issue, but we should be really careful to not make that situation worse.

Reply:
I'm thinking something like SyntacticEvent in React, where the underlaying raw event is still available, but there's a layer on top to smooth out the browser differences. (except we're smoothing out the builder differences)

Reply:
Yes, I think that's the plan for 6.3 (or whenever we add our next builder).

We probably shouldn't be exporting this function however, that was an oversight.